.github/workflows/build-test.yml

```diff
@@ -48,13 +48,7 @@ jobs:
       env:
         SUPER_SECRET: ${{ secrets.HF_HUB_TOKEN }}
       run: |
-        python -m pytest -s --cov=skops --cov-report=xml -v skops/
-
-    - name: Doc Tests
-      env:
-        SUPER_SECRET: ${{ secrets.HF_HUB_TOKEN }}
-      run: |
-        python -m pytest -s --cov=skops --doctest-modules skops/
+        python -m pytest -s --cov=skops --cov-report=xml --doctest-modules -v skops/
```

Comment:
Aren't the options redundant with here?

https://github.com/skops-dev/skops/blob/ca853d4d49c64484e5dcd7c49e1bc03e68b1a0ba/pyproject.toml#L9-L17

Reply:
Yes they are. I thought about removing the options there but then was unsure for the following reason: Is the main purpose of pyproject.toml to be for local development or for CI? If it's for the former, and we make a change that should only affect local development, we don't want to accidentally affect CI too. Therefore, I chose to have the options explicitly in CI (I assume that explicitly setting arguments takes precedence over the config file). If you think this distinction is not relevant, I can remove all redundant options.

Reply:
Interesting point of view. To me the point was to have it easier for people to have a local env more similar to the CI, so that the same tests are run with less effort. I personally added them there because I wanted to have them in the CI (definitely true for `--doctest-modules`).

Reply:
This is definitely a decision we can make -- running locally should be as close as possible to CI and the pyproject.toml should be the source of truth for that. It's just good to make that assumption explicit, so that we can be aware of that for the future.